content/new-in-22.md

```diff
@@ -374,8 +360,13 @@ Help users avoid and correct mistakes.
 <p><a href="https://www.w3.org/WAI/WCAG22/Understanding/redundant-entry">Understanding Redundant Entry</a></p>
 
 ### 3.3.8 Accessible Authentication (Minimum) (AA)
-<p class="persona"><a href="https://www.w3.org/WAI/people-use-web/user-stories/#supermarketassistant">Supermarket assistant</a> with cognitive disabilities:</p>
+<p>In brief:</p>
+   <ul>
+      <li><p><span class="whatwhy">What to do: </span>Don’t make people memorize or transcribe something to log in.</p></li>
+      <li><p><span class="whatwhy">Why it's important: </span>Some people with cognitive disabilities cannot memorize a username and password, or type them.</p></li>
```

Comment:
Since we say "memorize" under the what to do, does it add value saying "remember" here instead? I believe it is synonymously close enough that it adds value

```suggestion
      <li><p><span class="whatwhy">Why it's important: </span>Some people with cognitive disabilities cannot remember a username and password, or type them.</p></li>
```

Reply:
Or how about "recall"?